tests-functional/tests/test_eth_api.py

```diff
@@ -27,49 +32,92 @@ def validate_receipt(receipt, tx_hash, block_number, block_hash):
 @pytest.mark.rpc
 @pytest.mark.ethclient
 @pytest.mark.xdist_group(name="Eth")
-class TestEth(EthRpcSteps):
+# TODO:LOCALLY does not work; the method ethclient_transactionByHash does not exist/is not available, no option to extract tx details
```

Comment:
This is because it's part of private API, which we agreed to expose specifically for testing. To access it, you need to build `status-go` with `enable_private_api` flag, which is done on CI:
https://github.com/status-im/status-go/blob/46fd256209f3942aab024a4b9d0fe96a78891b27/_assets/scripts/run_functional_tests.sh#L43-L45

I guess we should add it to the docker build instructions. I can do it.

Reply:
totally makes sense, it was not obvious, I'll add it to doc and remove the comment from here